add cluster commands (#11743)
pingcap/docs: tiup/tiup-component-cluster-start.md

## Options

> ### --init

Reviewer: FYI: In [Step 7. Start a TiDB cluster](https://docs.pingcap.com/tidb/stable/production-deployment-using-tiup#step-7-start-a-tidb-cluster), this method is described as safe start and some descriptions are similar, especially the note.

> Starts the cluster in a secure way. It is recommended to use this option when the cluster is started for the first time. This method generates the password of the TiDB root user at startup and returns the password in the command line interface.

Author: ```suggestion
Starts the cluster in a safe way. It is recommended to use this option when the cluster is started for the first time. This method generates the password of the TiDB root user at startup and returns the password in the command line interface.
```

> **Note:**
>

> > - After using the secure startup method, you cannot log in to the database using the root user without a password. You need to record the password returned by the command line for subsequent operations.

Reviewer: ```suggestion
> - After safe start of a TiDB cluster, , you cannot log in to the database using the root user without a password. Therefore, you need to record the password returned by the command line for future logins.
```

> > - The automatically generated password is only returned once. If you do not record or forget the password, refer to [Forget the `root` password](/user-account-management.md#forget-the-root-password) to change the password.

Reviewer: ```suggestion
> - The password is generated only once. If you do not record or forget the password, refer to [Forget the `root` password](/user-account-management.md#forget-the-root-password) to change the password.
```
